atc/api/pipelines_test.go

```diff
@@ -1738,7 +1738,8 @@ var _ = Describe("Pipelines API", func() {
 							requestBody = `{"name":""}`
 						})
 
-						It("returns a warning in the response body", func() {
+						It("returns 400 Bad Request and an error in the response body", func() {
```

Comment:
Do we consider it a breaking change since the warning changed to a 400 ?

Reply:
I don't think so because it's the API. We don't support the API so the only clients we should worry about is fly and the web.